packages/ndla-ui/src/model/WordClass.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const WordClass = {
+  Noun: 'noun',
+  Verb: 'verb',
+  Pronoun: 'pronoun',
+  Determiner: 'determiner',
+  Adjective: 'adjective',
+  Adverb: 'adverb',
+  Preposition: 'preposition',
+  Conjunction: 'conjunction',
+  SubordinatingConjunction: 'subordinating conjunction',
+  Interjection: 'interjection',
+  Expression: 'expression',
+};
+
+export const WordClassChinese = {
+  Adjective: 'adjective',
+  Adverb: 'adverb',
+  ProperNoun: 'proper noun',
+  Auxiliary: 'auxiliary',
+  Complement: 'complement',
+  Conjunction: 'conjunction',
+  Coverb: 'coverb',
+  Quantifier: 'quantifier',
+  Marker: 'marker',
+  ModalVerb: 'modal verb',
+  MeasureWord: 'measure word',
+  Noun: 'noun',
```

Comment:
Bartek la inn denne som `noun-zh` i backend, så då kan du slå sammen disse to til en liste med kun WordClass. Det forplanter seg sjølvsagt til messages.